Expands the cloud native security doc section's landing page (#3309)
elastic/security-docs: docs/cloud-native-security/cloud-native-security-index.asciidoc

[[cloud-native-security-overview]]

= Cloud native security

> Elastic’s cloud security capabilities help you to improve your cloud security posture by comparing your cloud configuration to best practices, and allow you to monitor and investigate your Linux deployments inside and outside of Kubernetes.

Reviewer: It's better to say ```Elastic Security for Cloud``` rather than ```Elastic’s cloud security``` for consistency across docs, blogs and other materials.

> This page describes what each solution does and how to implement it.

Author: ```suggestion
This page describes what each solution does and provides links to more information.
```

[discrete]
== Cloud Security Posture Management (CSPM)
Cloud Security Posture Management helps you to harden your cloud security posture by assessing the configuration of your cloud accounts against security best practices defined by the Center for Internet Security (CIS). Once enabled, it runs regular scans which generate data that can inform your remediation efforts.

To learn more, refer to <<cspm, CSPM>>.

[discrete]
== Kubernetes Security Posture Management (KSPM)

> Kubernetes Security Posture Management helps you to harden the security posture of your Kubernetes deployments by assessing their configuration against security best practices defined by the Center for Internet Security (CIS). Once enabled it runs regular scans which generate data that can inform your remediation efforts.

Reviewer 2: ```suggestion
Kubernetes Security Posture Management helps you to harden the security posture of your Kubernetes deployments by assessing their configuration against security best practices defined by CIS. Once enabled, it runs regular scans which generate data that can inform your remediation efforts.
```

Author: I know we already defined CIS earlier on the page, but I think someone might skip the first section so it wouldn't hurt to define it here as well

To learn more, refer to <<kspm, KSPM>>.

[discrete]
== Cloud Native Vulnerability Management (CNVM)
Scans your cloud VMs for known vulnerabilities. When it finds a vulnerability, it supports your risk assessment by quickly providing information such as the CVE and CVSS, whether there is a known exploit of the vulnerability, and which software versions it affects.

To learn more, refer to <<vuln-management-overview, CNVM>>.

[discrete]
== Container Workload Protection
Provides cloud-native runtime protections for containerized environments by identifying and (optionally) blocking unexpected system behavior in Kubernetes containers. These capabilities are sometimes referred to as container drift detection and prevention. The solution also captures detailed process telemetry from monitored containers, allowing you to set up custom alerts and protection rules.

> To learn more, refer to <<d4c overview, Container workload protection>>.

Author: ```suggestion
To learn more, refer to <<d4c-overview, Container workload protection>>.
```

[discrete]
== Cloud Workload Protection (CWP)

> Cloud Workload Protection helps you monitor and protect your Linux VMs. It captures workload telemetry data containing process, file, and network activity. This data — along with Elastic's out-of-the-box detection rules and machine learning models — enables the automated identification of threats to your cloud VMs. Alerts based on these detections can reduce the time to identify and remediate threats.

Reviewer 2: ```suggestion
Cloud Workload Protection helps you monitor and protect your Linux VMs. It captures workload telemetry data containing process, file, and network activity. Use this data — along with Elastic's out-of-the-box detection rules and machine learning models — to automate the identification of threats to your cloud VMs. Alerts based on these detections can reduce the time to identify and remediate threats.
```

[[cloud-native-security-overview]]
= Cloud native security

> Elastic Security for Cloud helps you improve your cloud security posture by comparing your cloud configuration to best practices, and scanning for vulnerabilities. They also help you monitor and investigate your cloud workloads inside and outside Kubernetes.

Author: ```suggestion
Elastic Security for Cloud helps you improve your cloud security posture by comparing your cloud configuration to best practices, and scanning for vulnerabilities. It also helps you monitor and investigate your cloud workloads inside and outside Kubernetes.
```

To learn more, refer to <<cspm, Read the CSPM docs>>.

[discrete]
== Kubernetes Security Posture Management (KSPM)
Allows you to identify configuration risks in the various components that make up your Kubernetes cluster.
It does this by evaluating your Kubernetes clusters against secure configuration guidelines defined by the Center for Internet Security (CIS) and generating findings with step-by-step instructions for remediating potential security risks.

<<kspm, Read the KSPM docs>>.

[discrete]
== Cloud Native Vulnerability Management (CNVM)
Scans your cloud workloads for known vulnerabilities. When it finds a vulnerability, it supports your risk assessment by quickly providing information such as the vulnerability's CVSS and severity, which software versions it affects, and whether a fix is available.

<<vuln-management-overview, Read the CNVM docs>>.

[discrete]
== Container Workload Protection

> Provides cloud-native runtime protections for containerized environments by identifying and (optionally) blocking unexpected system behavior in Kubernetes containers. These capabilities are sometimes referred to as container drift detection and prevention. The solution also captures detailed process telemetry from monitored containers, allowing you to set up custom alerts and protection rules.

Reviewer 3: > The solution also captures detailed process telemetry

should read "...also captures detailed process and file..."

<<d4c-overview, Read the Container workload protection docs>>.

[discrete]
== Cloud Workload Protection for VMs

> Helps you monitor and protect your Linux VMs. It captures workload telemetry data containing process, file, and network activity. You can use this data — along with Elastic's out-of-the-box detection rules and machine learning models — to automate threat identification for your cloud VMs. Alerts that are generated from these detections help reduce the time to identify and remediate threats.

Reviewer 3: > It captures workload telemetry data containing process, file, and network activity.

I might suggest this change as this solution is built on Defend and thus has more capabilities than just streaming data:

> "It uses [Elastic Defend's endpoint capabilities](https://not-sure-what-link-is-best.com) to instantly detect and prevent malicious behavior and malware from impacting system security and captures workload telemetry data..."

Elastic's CSPM solution helps you to harden your cloud security posture by assessing the configuration of your cloud accounts against security best practices defined by the Center for Internet Security (CIS). Once enabled, it runs regular scans which generate data that can inform your remediation efforts.

To learn more, refer to <<cspm, CSPM>>.

[discrete]
== Kubernetes Security Posture Management (KSPM)
Elastic's KSPM solution helps you to harden the security posture of your Kubernetes deployments by assessing their configuration against security best practices defined by the Center for Internet Security (CIS). Once enabled it runs regular scans which generate data that can inform your remediation efforts.

To learn more, refer to <<kspm, KSPM>>.

[discrete]
== Cloud Native Vulnerability Management (CNVM)
Elastic's CNVM solution scans your cloud VMs for known vulnerabilities. When it finds a vulnerability, it supports your risk assessment by quickly providing information such as the CVE and CVSS, whether there is a known exploit of the vulnerability, and which software versions it affects.

To learn more, refer to [placeholder]

[discrete]
== Container Workload Protection (CWP)

> Elastic's CWP solution provides cloud-native runtime protections for containerized environments by identifying and (optionally) blocking unexpected system behavior in Kubernetes containers. These capabilities are sometimes referred to as container drift detection and prevention. Elastic CWP captures detailed process telemetry from monitored containers, allowing you to set up custom alerts and protection rules.

Author: So true, the twin acronyms are a killer. Great idea to just spell them out when possible
